LanguageFeatures/Inline-classes/static_analysis_member_invocation_A06_t01.dart

```diff
@@ -2,35 +2,29 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion We say that an inline class declaration DV has a member named n
-/// in the case where DV declares a member named n, and in the case where DV has
-/// no such declaration, but DV has a direct superinterface V that has a member
-/// named n. In both cases, the member declaration named n that DV has is said
-/// declaration.
+/// @assertion We say that an extension type declaration DV has a non-extension
+/// type member named n in the case where DV does not declare a member named n,
+/// but DV has a direct extension type superinterface V that has a non-extension
+/// type member named n, or DV has a direct non-extension type superinterface T
+/// whose interface contains a member signature named n.
 ///
-/// @description Checks that if inline class `V` has a superinterface with a
-/// member `m` then this member is also presents in `V`
+/// @description Checks that if an extension type `ET` has a superinterface with
+/// a member `m` then this member is also presents in `ET`
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=inline-class
 
-inline class V1 {
-  final int id;
-  V1(this.id);
-
+class I {
   void m1() {}
   int get m2 => 42;
   void set m3(int val) {}
 }
 
-inline class V implements V1 {
-  final int id;
-  V(this.id);
-}
+extension type ET(int id) implements I {}
```

Comment:
Again, `implements I` is a compile-time error because it doesn't satisfy `int <: I`. So we need to do something like `implements num`, and then we should be able to do `e.ceil()` where `e` has static type `ET`, but not `e.isEven`.